docs/content/2.guide/16.migrating.md

```diff
@@ -83,6 +83,38 @@ Deprecated for the same reason as the `localeLocation()` Option API.
 
 Use `localeHead()` instead for the Options API style.
 
+### Deprecated `baseUrl` component option
```

Comment:
We don't actually need to deprecate it, both would be valid and we can just push people to use site config instead. If they really want to use baseUrl then they should be free to.

Reply:
I removed the deprecations.